intel/reconcile.py

```diff
@@ -43,15 +43,29 @@ def reconcile(conf_dir, seconds, publish):
             k8sconfig.load_incluster_config()
             v1beta = k8sclient.ExtensionsV1beta1Api()
 
-            reconcile_report_type = third_party.ThirdPartyResourceType(
-                v1beta,
-                "cmk.intel.com",
-                "Reconcilereport")
-
-            node_name = os.getenv("NODE_NAME")
-            reconcile_report = reconcile_report_type.create(node_name)
-            reconcile_report.body["report"] = report
-            reconcile_report.save()
+            if k8s.get_kubelet_version(None) >= 7:
+                crdt = custom_resource.CustomResourceDefinitionType
+                reconcile_report_type = crdt(
+                    v1beta,
+                    "intel.com",
+                    "cmk",
+                    ["cmk"]
+                )
+
+                node_name = os.getenv("NODE_NAME")
```

Comment:
Why do you here extract `node_name` to a separate variable and in `intel/nodereport.py` file not? Please be consistent.